src/main/java/seedu/address/commons/util/TagUtil.java

```diff
@@ -0,0 +1,132 @@
+package seedu.address.commons.util;
+
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Optional;
+import java.util.Set;
+
+import seedu.address.model.person.Address;
+import seedu.address.model.person.Birthday;
+import seedu.address.model.person.Email;
+import seedu.address.model.person.Name;
+import seedu.address.model.person.Phone;
+import seedu.address.model.tag.Tag;
+
+public class TagUtil {
```

Comment:
Not sure if calling it `TagUtil` is a good name since it's not only limited to `Tag `and `Untag`, `Edit` uses it too.

Reply:
makes sense, ill change this